numpy/random/_generator.pyx

```diff
@@ -665,6 +665,13 @@ cdef class Generator:
         array([3,1,0]) # random
         >>> #This is equivalent to rng.permutation(np.arange(5))[:3]
 
+        Generate a uniform random sample from a 2-D array, without
+        replacement:
+
+        >>> rng.choice([[0, 1, 2], [3, 4, 5], [6, 7, 8]], 2, replace=False)
+        array([[3, 4, 5],
+               [0, 1, 2]])
+
```

Comment:
This example is raising a documentation error.
```
File "build/testenv/lib/python3.7/site-packages/numpy/random/_generator.cpython-37m-darwin.so", line ?, in Generator.choice
Failed example:
    rng.choice([[0, 1, 2], [3, 4, 5], [6, 7, 8]], 2, replace=False)
Expected:
    array([[3, 4, 5],
           [0, 1, 2]])
Got:
    array([[6, 7, 8],
           [3, 4, 5]])
```
You need to use a seed to make it repeatable, or add `# may vary` to the first output line. Grep to see examples.

Reply:
Ah, it had occurred to me, but I guess I missed the `# random` in the other examples and couldn't find anything about it in the developer documentation. Cheers!